framework/wazuh/core/wlogging.py

```diff
@@ -48,20 +48,31 @@ def computeArchivesDirectory(self, rotated_filepath):
         rotated_file = os.path.basename(rotated_filepath)
         year, month, day = re.match(r'[\w\.]+\.(\d+)-(\d+)-(\d+)', rotated_file).groups()
         month = calendar.month_abbr[int(month)]
-
-        log_path = os.path.join(self.baseFilename.replace('.log', ''), year, month)
+        log_path = os.path.join(os.path.splitext(self.baseFilename)[0], year, month)
         if not os.path.exists(log_path):
             utils.mkdir_with_mode(log_path, 0o750)
 
-        return f'{log_path}/{os.path.basename(self.baseFilename).replace(".log", "")}-{day}.log.gz'
+        return f'{log_path}/{os.path.basename(self.baseFilename)}-{day}.gz'
+
+
+class CustomFilter():
+    """
+    Defines a custom filter to differentiate between log types
+    """
+    def __init__(self, log_type):
+        self.log_type = log_type
+
+    def filter(self, record):
+        # If the log file is not specifically filtered, then It should log into both files

```

Comment:
Add docstring to these 2 methods, review this comment to (`then It...`, why caps?)